### PR TITLE
Add FAROUT token

### PR DESCRIPTION
**Pull request by jejdouay**
## I agree to not ping *anybody* on Discord/Twitter/email about this pull request. Instead I will inquire by posting a new comment in the pull request if needed.
----
PRs are reviewed in bulk and and can take up to **two weeks** to be merged.  

**Please provide the following information for your token.**

Please include change to the `src/tokens/solana.tokenlist.json` file in the PR. 
DON'T modify any other token on the list.

At minimum each entry should have
* Token Address: 
J5gLhk6mmQ4PSoir1Ufh8JY2ytEHA93YupzYiTFVCgcL
* Token Name: 
Far-Out
* Token Symbol : 
FAROUT
* Logo: (logo should be uploaded under assets/mainnet/<mint address>/*.<png/svg>)
assets/mainnet/J5gLhk6mmQ4PSoir1Ufh8JY2ytEHA93YupzYiTFVCgcL/logo.png
* Link to the official homepage of token:
http://far-out.biz
* Coingecko ID if available (https://www.coingecko.com/api/documentations/v3#/coins/get_coins__id_):
NA

### CODE REVIEW

File: src/tokens/solana.tokenlist.json [hunk 1]
```diff
@@ -14059,11 +14059,20 @@
...
       }
+    },
+    {
+      "chainId": 101,
+      "address": "J5gLhk6mmQ4PSoir1Ufh8JY2ytEHA93YupzYiTFVCgcL",
+      "symbol": "FAROUT",
+      "name": "Far-Out Token",
+      "decimals": 8,
+      "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/assets/J5gLhk6mmQ4PSoir1Ufh8JY2ytEHA93YupzYiTFVCgcL/logo.png",
+      "tags": []
```
Comment (bartosz-lipinski) [inline src/tokens/solana.tokenlist.json:14070]: please add links to social and website.

Comment (jejdouay) [reply src/tokens/solana.tokenlist.json]: Updated what I can today. Was not expecting to be required to supply these information at this point.

**Comment by jejdouay** [comment]
Have I missed something ?

**Comment by jejdouay** [comment]
@rmshea please review. Waiting for this pull request to open market on serum and start operations. Thank you !

**Comment by jejdouay** [comment]
nothign happening creating a new one !

**Pull request closed without merging**